R/aa_utils.R

```diff
@@ -12,11 +12,22 @@ NULL
 .drive <- new.env(parent = emptyenv())
 .drive$base_url_files_v3 <- "https://www.googleapis.com/drive/v3/files"
 .drive$base_url <- "https://www.googleapis.com"
+.drive$mimetypes <-
+  system.file("extdata", "mimetypes.csv", package = "googledrive") %>%
+  read.csv(stringsAsFactors = FALSE)
```

Comment:
You don't mean to keep my toy example in here, do you? If not, these lines and the file they load should go away.